languages.yaml

```diff
@@ -28,10 +28,10 @@ Ada:
       - cyan
       - blue
     hex:
-      - '#FFFFFF'
-      - '#0018C9'
-      - '#0C0A7C'
-    chip: '#02F88C'
+      - "#FFFFFF"
```

Comment:
Your linting tool seems to have replaced all single quotes with double quotes.
Please revert this.

Reply:
Good catch! I'm on it